acceptance/framework/consul/helm_cluster.go

```diff
@@ -201,71 +201,68 @@ func (h *HelmCluster) Destroy(t *testing.T) {
 			}
 		}
 	}
-
-	// Verify all Consul Pods are deleted.
-	pods, err = h.kubernetesClient.CoreV1().Pods(h.helmOptions.KubectlOptions.Namespace).List(context.Background(), metav1.ListOptions{LabelSelector: "release=" + h.releaseName})
-	require.NoError(t, err)
-	for _, pod := range pods.Items {
-		if strings.Contains(pod.Name, h.releaseName) {
-			t.Logf("Found pod which should have been deleted: %s", pod.Name)
-			t.Fail()
+	// Retry a few times because sometimes certain resources (like PVC) take time to delete
+	// in cloud providers.
+	retry.RunWith(&retry.Counter{Wait: 1 * time.Second, Count: 10}, t, func(r *retry.R) {
```

Comment:
This line is the only thing new here.
All of the other changes just change the 
`t.Fail()` to `r.Errorf()`